cypress/page/shipping-step.page.ts

```diff
@@ -4,7 +4,7 @@ class ShippingStepPage {
   private Checkout: string;
 
   constructor() {
-    this.AgreeTerms = "div.checker input";
+    this.AgreeTerms = "div.checker input[name='cgv']";
```

Comment:
this element has an id, you can use it instead of the name.

Reply:
👍🏼